CHANGELOG.md

```diff
@@ -8,6 +8,19 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.4] - 2023-02-02
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.3...v0.3.4)
+
+### Fixed
+
+* Fixed environment variable mapping (#498). For the full list of environment variables see [.config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
+
+### Removed
+
+* The ReadTuples endpoint has been removed. Please use [Read](https://openfga.dev/api/service#/Relationship%20Tuples/Read) with no tuple key instead. (#495)
```

Comment:
Can we put the 'e.g.' bit on a newline so it renders more nicely.

* blah...

  e.g. `POST /stores/<store_id>/read` with `{}` as the body